.travis.yml

```diff
@@ -1,4 +1,8 @@
-language: java
+language: android
```

Comment:
Seems like it; I believe that it's just a normal Java environment *plus* the Android SDK components. Anyway the build seems to work in Travis, so... /shrug